package/MDAnalysis/analysis/density.py

```diff
@@ -501,6 +501,7 @@ def current_coordinates():
     pm = ProgressMeter(u.trajectory.n_frames, interval=interval, quiet=quiet,
                        format="Histogramming %(n_atoms)6d atoms in frame "
                        "%(step)5d/%(numsteps)d  [%(percentage)5.1f%%]\r")
+    start, stop, step = u.trajectory.check_slice_indices(start, stop, step)                    
```

Comment:
This reproduce how slicing works everywhere in Python. If start, end, or step is `None`, then the correct value will be deduced based on the rest. Most of the time, start will be automatically set to 0, end to `len(u.trajectory)`, and step to 1. But if you want to read your whole trajectory backward, then you will set step to -1, you will left start and end to `None`, and their correct value will be deduced.

Look at how `range` works. It is the same here.